codebuild/cd/musl-linux-build.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -ex
+
+IMAGE_NAME=$1
+shift
+CLASSIFIER=$1
+shift
+
+# Pry the builder version this CRT is using out of ci.yml
+BUILDER_VERSION=$(cat .github/workflows/ci.yml | grep 'BUILDER_VERSION:' | sed 's/\s*BUILDER_VERSION:\s*\(.*\)/\1/')
+echo "Using builder version ${BUILDER_VERSION}"
+
+aws ecr get-login-password | docker login 123124136734.dkr.ecr.us-east-1.amazonaws.com -u AWS --password-stdin
+export DOCKER_IMAGE=123124136734.dkr.ecr.us-east-1.amazonaws.com/${IMAGE_NAME}:${BUILDER_VERSION}
+export QEMU_IMAGE=123124136734.dkr.ecr.us-east-1.amazonaws.com/multiarch-qemu-user-static:latest
```

Comment:
Docker has a pull limit for users, and we have started running into that limit. So, we are using one from ECR instead.